src/app/utils/error-handler.ts

```diff
@@ -11,8 +12,9 @@ import {
   SERVER_ERROR,
 } from '../constants/http-constants';
 
-export function handleApiValidationError(error: unknown) {
-  console.error('API route error:', error);
+
+export function handleApiValidationError(error: unknown, logger:LogHandler) {
+  logger.error(`API route error: ${error}`);
```

Comment:
Suggestion: we can supply the error stack trace as an additional argument to the error method instead of putting the error as a string value in the message
```suggestion
  logger.error('API route error', error);
```